src/pages/User/components/PullRequests/index.js

```diff
@@ -8,6 +8,39 @@ import PullRequest from './PullRequest';
 import IssuesLink from './IssuesLink';
 import MeLinkInfo from './MeLinkInfo';
 import { GITHUB_TOKEN } from '../../../../config';
+import NotAMember from '../Modal/NotAMember';
+
+const GITHUB_ORG_NAME = 'leapfrogtechnology';
+
+/**
+ * returns an object containing user info
+ *
+ * @param {string} username
+ * @returns {*}
+ */
+export async function fetchUserInfo(username) {
+  const apiUrl = [
+    `https://api.github.com/search/issues?q=author:${username}+is:pr+created:2019-10-01..2019-10-31`,
+    `https://api.github.com/search/users?q=user:${username}`,
+    `https://api.github.com/orgs/${GITHUB_ORG_NAME}/members/${username}`
+  ];
+
+  const allResponses = apiUrl.map(url =>
+    fetch(url, {
```

Comment:
Extract this as a separate function `fetchInfoFromGitHub(url, token)`. It could be as simple as this:
```js
const result = apiUrls.map(url => fetchInfoFromGitHub(url, GITHUB_TOKEN))
```